media/js/base/search-params.js

```diff
@@ -11,7 +11,7 @@ var _SearchParams = function (search) {
     search = search.match(/^\??(.*)/)[1];
     search = search ? search.split(/[&;]/m) : [];
 
-    $.each(search, function (index, param) {
+    search.forEach(function (param) {
```

Comment:
Looking in our bundle definitions, this file is [included in our legacy IE8 bundle](https://github.com/mozilla/bedrock/blob/master/media/static-bundles.json#L1527). According to [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach), `Array.forEach` is only supported in IE9, so we may need to support even more ancient machinery here :(